pkg/services/ngalert/provisioning/contactpoints.go

```diff
@@ -540,22 +538,10 @@ func RemoveSecretsForContactPoint(e *apimodels.EmbeddedContactPoint) (map[string
 	return s, nil
 }
 
-// handleWrappedError unwraps an error and wraps it with a new expected error type. If the error is not wrapped, it returns just the expected error.
-func handleWrappedError(err error, expected error) error {
```

Comment:
NOTE: No longer necessary since we moved to errorutils in `receiver_scv` responses.